GameData/RealismOverhaul/Engine_Configs/Electric_Engine_Configs/MR510_Config.cfg

```diff
@@ -87,6 +87,12 @@
 				ratio = 1.0
 				DrawGauge = True
 			}
+			PROPELLANT
+			{
+				name = Helium
+				ratio = 18.6
+				ignoreForIsp = True
+			}
```

Comment:
Note: Adding the resource req is breaking for existing parts

Reply:
I’m fairly certain no part actually uses the MR-510 confit at the moment, but this could be moved.